### PR TITLE
week-01 homework part1

### PR DESCRIPTION
**Pull request by AtomicZero**
Homework Assignment:

To update company's website as per requirements detailed in brief.
Changes/updates to code include the following:

- updated HTM title to show company name Horiseon
- updated HTML tags in document head and body to semantic tags for SEO compliance as per https://developer.mozilla.org/en-US/docs/Web/HTML/Element
- updated image tags with ALT descriptions for screen readers apps to help the visually impaired
- updated CSS Stylesheet to correspond with semantic tags and appropriately grouped per each section

Documents included in upload:

1. Updated HTML doc
2. Updated CSS stylesheet
3. Changelog text

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,82 +1,101 @@
...
+    <title>Horiseon</title>
 </head>
 
+
+
+
 <body>
-    <div class="header">
+
+    <HEADER>
```
Comment (surajverma2587) [inline index.html]: Please note that all HTML elements are lowercase. In this case you have used the correct semantic HTML element but have the casing wrong. Please correct.

File: index.html [hunk 2]
```diff
@@ -1,82 +1,101 @@
...
+                    </li>
+                    <li>
+                        <a href="#social-media-marketing">Social Media Marketing</a>
+                    </li>
+                </ul>
+            </nav>
+    </HEADER>
+
+
+    <div class="hero">
```
Comment (surajverma2587) [inline index.html]: Research the attribute called `title` and see if you can add a `title` attribute to the `div`. Also, think of why a `title` attribute could be useful for this `div`.

[title](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title)

**HINT:** tooltip

File: assets/css/style.css [hunk 3]
```diff
@@ -8,39 +10,43 @@ body {
     background-color: #d9dcd6;
```
Comment (surajverma2587) [inline assets/css/style.css]: Maybe worth investigating if we can add a `font-family` here if it being used across the web page and for cases when they are different we can overwrite them in the respective styles.

File: index.html [hunk 4]
```diff
@@ -1,82 +1,101 @@
...
+                    Social media continues to have a sizable influence on buying habits. Social media marketing helps you determine which platforms are suited to your brand, using analytics to find the right markets and increase your lead generation.
+                </p>
+        </section>
+    </main>
+      
+    
+
+
+    <aside class="benefits">
         <div class="benefit-lead">
```
Comment (surajverma2587) [inline index.html]:
```suggestion
        <div class="benefit">
```
If you notice in the stylesheet there is duplication of the styles applied for the 3 benefits. They could all be classed a single style declaration with a class name like `benefit` or any name of your choosing. Code duplication is not preferred and for this task it is important to consider minimising obvious duplication of code. Please fix this and also make the relevant changes in the style sheet.

File: index.html [hunk 5]
```diff
@@ -1,82 +1,101 @@
...
+
+
+    <div class="hero">
+
     </div>
-    <div class="benefits">
+
+
+    <main>
+        <section id="search-engine-optimization" class="service">
```
Comment (surajverma2587) [inline index.html]: You have assigned a class called `service` here but that isn't being used in the stylesheet. In the stylesheet there is obvious duplication of styling for these sections which can all be grouped to a single css selector of `service`. Please make the change on the css file to reduce code duplication. Code duplication is a requirement for this assignment and should be addressed.

Comment (AtomicZero) [reply index.html]: Yeah I do recall seeing that one somewhere and thought it was a special tag for some accessibility app or something. so I put that one in right away. Now I realize it's a CSS tag  :P

File: index.html (hunk 1 above)
Comment (AtomicZero) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (AtomicZero) [reply index.html]: done

File: assets/css/style.css (hunk 3 above)
Comment (AtomicZero) [reply assets/css/style.css]: done

File: index.html (hunk 4 above)
Comment (AtomicZero) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (AtomicZero) [reply index.html]: done

File: assets/css/style.css [hunk 6]
```diff
@@ -6,41 +7,44 @@
 
 body {
     background-color: #d9dcd6;
+    font-family: 'Courier New', Courier, monospace;
```
Comment (surajverma2587) [inline assets/css/style.css]:
```suggestion
    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```

I would set this as the default `font-family` and remove the `font-family` from `#main-panel`, `.benefits` and `header nav`. In other cases where you have to overwrite the `font-family` like `header` and `footer` you leave the other `font-family` there which is `'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif`

File: assets/css/style.css [hunk 7]
```diff
@@ -6,41 +7,44 @@
...
+header h1 .seo {
     color: #d9dcd6;
 }
 
-.header div {
+header nav {
     padding-top: 15px;
     margin-right: 20px;
     float: right;
     font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```
Comment (surajverma2587) [inline assets/css/style.css]: You can remove this if you make this the default `font-family` in the `body` tag

File: assets/css/style.css [hunk 8]
```diff
@@ -86,55 +94,26 @@ p {
     background-color: #2589bd;
```
Comment (surajverma2587) [inline assets/css/style.css]: You can remove the `font-family` here if you declare the default `font-family` in the `body` tag

File: assets/css/style.css [hunk 9]
```diff
@@ -143,58 +122,25 @@ p {
     color: #ffffff;
```
Comment (surajverma2587) [inline assets/css/style.css]: You can remove this if you make this the default `font-family` in the `body` tag

**Pull request merged**